pkg/process/collector.go

```diff
@@ -0,0 +1,121 @@
+package process
```

Comment:
**question:** This package isn't scoped to a particular cortex-module. In the past packages of this nature have been stored in`pkg/util/*`. Any reason you chose not to? IMO reducing the usage of the util package as a catch all would lead us towards more discrete package design overall.

Reply:
Right. Moved to `pkg/util/process`.